web/.babelrc

```diff
@@ -1,3 +1,4 @@
 {
-    "presets": ["@babel/preset-env", "@babel/preset-react", "@linaria"]
+    "presets": ["@babel/preset-env", "@babel/preset-react", "@linaria"],
```

Comment:
I should have ask this on previous PR.
does next js requires explicit babelrc configuration?

https://nextjs.org/docs/advanced-features/customizing-babel-config

Reply:
Please wait, I'll double-check the environment so we can reduce presets

Reply:
I just recognized that we can reuse `next/babel` like the [docs](https://nextjs.org/docs/advanced-features/customizing-babel-config).
I'll fix it, but please mind that we still need the `.babelrc` for linaria(extracting the CSS at build time)

Reply:
Alright, after next commit, you can merge this yourself.